### PR TITLE
Expose db_pool_max and db_pool_init attrbiutes for bookshelf like oc_…

### PR DESCRIPTION
**Pull request by itmustbejj**
…bifrost and erchef

Signed-off-by: Josh Hudson <jhudson@chef.io>

### CODE REVIEW

**Comment by stevendanna** [comment]
Looks good to me, not sure why we didn't do this in the initial change.  While we are here, should we do the same thing over in the bookshelf configuration as well?

File: omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb [hunk 1]
```diff
@@ -114,8 +114,8 @@
  {pooler, [
            {pools, [[{name, sqerl},
                      {max_count, <%= @db_pool_size %> },
-                     {init_count, <%= @db_pool_size %> },
-                     {queue_max, <%= @db_pool_queue_max %>},
+                     {init_count, <%= @db_pool_init || @db_pool_size %> },
+                     {queue_max, <%= @db_pool_max || @db_pool_queue_max %>},
```
Comment (irvingpop) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb]: shouldn't change this one,  but line 116 above (max_count)

**Comment by itmustbejj** [comment]
Found a typo, please don't merge.

File: omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb (hunk 1 above)
Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/bookshelf.config.erb]: Good catch!

**Comment by irvingpop** [comment]
👍 looks good now

**Pull request merged**